client/src/components/UI/card/JobCard.module.css

```diff
@@ -0,0 +1,153 @@
+.matches-tag {
+  background-color: rgb(53, 188, 50);
+  color: #fff;
+  font-size: 1.4rem;
+  border-radius: 1.5rem;
+  margin-bottom: 5.5rem;
+  margin-left: 2rem;
+  text-align: center;
+  max-width: 10rem;
+  overflow: hidden;
+  white-space: nowrap;
+  padding: 0.1875rem 0.5rem;
+  font-weight: bold;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  top: 0.6rem;
+  transform: translateX(0);
+  transform: translateY(0);
+  width: 100%;
+}
+
+.company-info {
+  display: flex;
+  flex-direction: column;
+}
+
+.job-header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.card-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.go-to-talents {
+  width: 35rem;
+  height: 40px;
+  padding: 0px, 0px, 3px, 0px;
+  border-radius: 1.2rem;
+  border: 20px;
+  border-color: rgba(89, 89, 89, 1);
+  font-size: 2rem;
+  font-weight: 700;
+  text-align: center;
+}
+
+.avatar {
+  width: 80px;
+  height: 80px;
+  border-radius: 4px;
+  margin-right: 16px;
+}
+
+.card-content div:nth-child(2) {
+  flex: 1;
+}
+
+.job-info {
+  display: flex;
+}
+
+.card {
+  box-shadow: 0.2rem 0.4rem 0.4rem rgba(0, 0, 0, 0.1);
+  width: 40rem;
+  margin: 2rem 4rem 2rem 0;
+}
+
+h2,
+h3,
+h4,
+h5 {
```

Comment:
avoid giving style to elements, use only classes, otherwise the main advantage of using css modules (have styles contained to components) is lost. 
I saw you already gave classes to these elements, so the styling should be there.

Reply:
replaced it